unstract/connectors/src/unstract/connectors/databases/oracle-db/oracle_db.py

```diff
@@ -12,12 +12,12 @@ class OracleDB(UnstractDB):
     def __init__(self, settings: dict[str, Any]):
         super().__init__("OracleDB")
 
-        self.config_dir = settings["config_dir"]
-        self.user = settings["user"]
-        self.password = settings["password"]
-        self.dsn = settings["dsn"]
-        self.wallet_location = settings["wallet_location"]
-        self.wallet_password = settings["wallet_password"]
+        self.config_dir = settings.get("config_dir", None)
+        self.user = settings.get("user", None)
+        self.password = settings.get("password", None)
+        self.dsn = settings.get("dsn", None)
+        self.wallet_location = settings.get("wallet_location", None)
+        self.wallet_password = settings.get("wallet_password", None)
```

Comment:
NIT: Try to use the same defaults mentioned in JSON schema for correctness - this can later be optimized by creating the `settings` dict using JSON schema methods to generate it with defaults